qcoptions.py

```diff
@@ -8,7 +8,7 @@
                   help="Minimum value of scan parameter", metavar="scanmin", default=0)
 parser.add_option("--scanmax", type="int", dest="scanmax",
                   help="Maximum value of scan parameter", metavar="scanmax", default=254)
-parser.add_option("--vfatmask", type="int", dest="vfatmask",
+parser.add_option("--vfatmask", type="int", dest="vfatmask", 
```

Comment:
could be the way that github manages comments linked to code, as i had a similar issue just now where what i see on the PR dashboard and what I see in the latest version of the file in the PR aren't in agreement...
UPDATE nope, just checked and confirm what you've seen, could be that it's a single space, or a tab (not sure what `vim` is showing you?